### PR TITLE
Updated AddThisBasicToolbox to render proper markup.

### PR DESCRIPTION
**Pull request by doylejd**
- Added preRender functions back to AddThisWrapper, AddThisElement to handle things that were not being picked up OOTB. 
- Updated twig templates to render proper markup.

### CODE REVIEW

**Comment by doylejd** [comment]
@damontgomery  Can you take a look at this real quick? This has the proper markup rendering, but there are a couple things that didn't work OOTB that I think should. They are listed in the TODO: section of the PR.

**Comment by doylejd** [comment]
@damontgomery  This is ready to be looked at again.

File: src/Element/AddThisElement.php [hunk 1]
```diff
@@ -20,9 +20,12 @@ class AddThisElement extends RenderElement {
    * {@inheritdoc}
    */
   public function getInfo() {
+    $class = get_class($this);
     return [
       '#theme' => 'addthis_element',
+      '#attributes' => array(),
```
Comment (damontgomery) [inline src/Element/AddThisElement.php]: `'#attributes' => [];` is the recommended syntax.

**Comment by damontgomery** [comment]
@doylejd, I think for now this should be good.

I would recommend the switch statement logic be moved to a plugin system, but that could be done later.

I would recommend that the wrapper and element render elements be removed and to use a single element that is `addthis_basic_toolbox` and which takes a list of items. I'd put all the foreach logic into the twig template. These two element types are only used once and I can't see a use case for changing the markup elements (div / a / etc) unless it was being done site-wide in which case the pattern is to override the twig template. Feeding frontend config through the backend never works in my opinion.

**Comment by doylejd** [comment]
@damontgomery Thanks for the feedback - your suggestions have been moved into the issue queue. Closing this out.

**Pull request merged**